sdk/azcore/runtime/poller.go

```diff
@@ -50,8 +50,14 @@ const (
 // NewPollerOptions contains the optional parameters for NewPoller.
 type NewPollerOptions[T any] struct {
 	// FinalStateVia contains the final-state-via value for the LRO.
+	// NOTE: used only for Azure-AsyncOperation and Operation-Location LROs.
 	FinalStateVia FinalStateVia
 
+	// OperationLocationResultPath contains the JSON path to the result's
+	// payload when it's included with the terminal success response.
+	// NOTE: only used for Operation-Location LROs.
+	OperationLocationResultPath string
```

Comment:
The name is long and I'd like to shorten it to `ResultPath`. However, since it's only applicable to `Operation-Location` LROs having the prefix does clarify the usage. Maybe the doc comment is sufficient? We already have a similar problem with `FinalStateVia`.

Reply:
Will leave with the long name as this field is specific to `Operation-Location`.